src/modules/vtol_att_control/vtol_type.h

```diff
@@ -207,6 +211,9 @@ class VtolType
 
 	virtual void parameters_update() = 0;
 
+	void setDt(float dt) {_dt = dt; }
```

Comment:
documentation with brief, `param[in]` and units pls.

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/98ea2aeb1c80f716c5335c194fbe02116df85ca6